mindmeld/converter/converter.py

```diff
@@ -62,20 +62,6 @@ def create_directory(directory):
             except OSError:
                 logger.error("Cannot create directory at %s", directory)
 
-    @staticmethod
```

Comment:
We should keep this function since it is still being used by the Rasa converter.